modules/apps/27-interchain-accounts/keeper/relay_test.go

```diff
@@ -91,3 +100,128 @@ func (suite *KeeperTestSuite) TestTrySendTx() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestOnRecvPacket() {
+	var (
+		path       *ibctesting.Path
+		msg        sdk.Msg
+		txBytes    []byte
+		packetData []byte
+		sourcePort string
+	)
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"Interchain account successfully executes banktypes.MsgSend", func() {
+				// build MsgSend
+				amount, _ := sdk.ParseCoinsNormalized("100stake")
+				interchainAccountAddr, _ := suite.chainB.GetSimApp().ICAKeeper.GetInterchainAccountAddress(suite.chainB.GetContext(), path.EndpointA.ChannelConfig.PortID)
+				msg = &banktypes.MsgSend{FromAddress: interchainAccountAddr, ToAddress: suite.chainB.SenderAccount.GetAddress().String(), Amount: amount}
+				// build packet data
+				txBytes, err := suite.chainA.GetSimApp().ICAKeeper.SerializeCosmosTx(suite.chainA.Codec, msg)
+				suite.Require().NoError(err)
+
+				data := types.IBCAccountPacketData{Type: types.EXECUTE_TX,
+					Data: txBytes}
+				packetData = data.GetBytes()
+			}, true,
+		},
+		{
+			"Cannot deserialize txBytes", func() {
```

Comment:
another minor nit: I think the usual standard is to keep testcase names in lowercase. Or at least that's what I'm generally used to seeing

Reply:
I'm a fan of lowercase, but I don't think there's a good reason most tests are lowercase (aside from me probably having written them). Open to changing, but otherwise agree best to stick to what's already used

Reply:
Thanks. Updated :+1: